src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+is a platform that enables you to programmatically author, schedule, and monitor workflows. Using Airflow,
+you can build a workflow for SageMaker training, hyperparameter tuning, batch transform and endpoint deployment.
+You can use any SageMaker deep learning framework or Amazon algorithms to perform above operations in Airflow.
+
+There are two ways to build SageMaker workflow. Using Airflow SageMaker operators or using Airflow PythonOperator.
+
+1. SageMaker Operators: Since Airflow 1.10.1, we contributed special operators just for SageMaker operations.
+Each operator takes a configuration dictionary that defines the corresponding operation. And we provide APIs to
+generate the configuration dictionary in SageMaker Python SDK. Current supported SageMaker operators are:
+
+* ``SageMakerTrainingOperator``
+* ``SageMakerTuningOperator``
+* ``SageMakerModelOperator``
+* ``SageMakerTransformOperator``
+* ``SageMakerEndpointConfigOperator``
+* ``SageMakerEndpointOperator``
+
+2. PythonOperator: Airflow built-in operator that could execute Python callables. You could use SageMaker Python SDK to
+customize your workflow with PythonOperator.
+
+Using Airflow on AWS
+~~~~~~~~~~~~~~~~~~~~
+
+Turbine is an open source AWS CloudFormation template to create Airflow resources stack on AWS.
+You can get it here: https://github.com/villasv/aws-airflow-stack
+
+Using Airflow SageMaker Operators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from Airflow 1.10.1, you could use SageMaker operators in Airflow. All SageMaker operators take a
+configuration dictionary that can be easily generated by SageMaker Python SDK. For example:
+
+.. code:: python
+
+    import sagemaker
+    from sagemaker.tensorflow import TensorFlow
+    from sagemaker.workflow.airflow import training_config, transform_config_from_estimator
+
+    estimator = TensorFlow(entry_point='tf_train.py',
+                           role='sagemaker-role',
+                           framework_version='1.11.0',
+                           training_steps=1000,
+                           evaluation_steps=100,
+                           train_instance_count=2,
+                           train_instance_type='ml.p2.xlarge')
+
+    # train_config specifies SageMaker training configuration
+    train_config = training_config(estimator=estimator,
+                                   inputs=your_training_data_s3_uri)
+
+    # trans_config specifies SageMaker batch transform configuration
+    trans_config = transform_config_from_estimator(estimator=estimator,
+                                                   instance_count=1,
+                                                   instance_type='ml.m4.xlarge',
+                                                   data=your_transform_data_s3_uri,
+                                                   content_type='text/csv')
+
+Now we can pass these configurations to related SageMaker operators and create the workflow:
```

Comment:
Now you can pass these configurations to the corresponding SageMaker operators and create the workflow:

Reply:
Updated.